### PR TITLE
Block pagination for new blocks

### PR DESCRIPTION
**Pull request by robin-thomas**

### CODE REVIEW

**Comment by vgrichina** [comment]
Your PR Server URL is https://near-explorer-frontend-pr-52.onrender.com.

Follow its progress at https://dashboard.render.com/web/srv-bl0ddvb60hn04ibd8q60.

Cheers,

[The Render Team](https://render.com)

**Comment by frol** [comment]
Is it WIP?

File: frontend/components/api/Blocks.js [hunk 1]
```diff
@@ -101,6 +101,49 @@ const Blocks = {
       console.error(error);
       throw error;
     }
+  },
+
+  getNextBlocks: async (lastBlockHeight, limit = 15) => {
```
Comment (frol) [inline frontend/components/api/Blocks.js]: Am I reading this right that you basically implement a class/namespace for the functions? Why do you need the `const Blocks`? Can these be plain functions or defined as methods of a `Blocks` class?

**Comment by frol** [comment]
Staled

**Pull request closed without merging**